lib/util/git/index.ts

```diff
@@ -361,23 +362,34 @@ export async function checkoutBranch(branchName: string): Promise<CommitSha> {
   }
 }
 
+async function fileList(
+  directory: string,
+  excludes: string[] = []
+): Promise<string[]> {
+  let list: string[] = [];
+
+  const files = (await fs.readdir(directory)).filter((file) =>
+    excludes.every(
+      (exclude) => file !== exclude && !file.startsWith(`${exclude}/`)
```

Comment:
I have a doubt in my code now. This may not filter correctly is the submodule is nested deeper than the top level directory